example/host_translations.csv

```diff
@@ -0,0 +1,55 @@
+ipaddr,hostname
```

Comment:
Per our conversation in Slack, yes, this csv is used by a rake task to populate Redis.